config/Config.h

```diff
@@ -29,6 +29,7 @@ class Config : public ConfigReader
     CFG_PROP(int32_t, silenceThresholdLevel, 127);
     CFG_PROP(uint32_t, lastN, 3);
     CFG_PROP(uint32_t, lastNextra, 2);
+    CFG_PROP(uint32_t, activeTalkerDbDiff, 18);
```

Comment:
Maybe DbDiff is not saying much. Diff to what?
activeTalkerSilenceThresholdDb

Reply:
Changed.